frmts/georaster/CMakeLists.txt

```diff
@@ -13,13 +13,19 @@ if (GDAL_ENABLE_DRIVER_JPEG AND GDAL_USE_JPEG_INTERNAL)
   target_compile_definitions(gdal_GEOR PRIVATE -DJPEG_SUPPORTED)
 endif ()
 
-gdal_target_link_libraries(gdal_GEOR PRIVATE Oracle::OCI)
-
+# Include libjpeg first so that if we want to use jpeg-turbo on homebrew
+# /usr/local/opt/jpeg-turbo gets included first, before /usr/local/opt
+# that contains libjpeg 9e
 if (GDAL_ENABLE_DRIVER_JPEG AND (NOT GDAL_USE_JPEG_INTERNAL))
   target_compile_definitions(gdal_GEOR PRIVATE -DJPEG_SUPPORTED)
   gdal_target_link_libraries(gdal_GEOR PRIVATE JPEG::JPEG)
+  if (EXPECTED_JPEG_LIB_VERSION)
+    target_compile_definitions(gdal_GEOR PRIVATE EXPECTED_JPEG_LIB_VERSION=${EXPECTED_JPEG_LIB_VERSION})
```

Comment:
Instead of repeating this whenever `JPEG::JPEG` is used, I wonder if it would have been easier to add this early to `JPEG::JPEG`'s `INTERFACE_LINK_OPTIONS`. However, this needs CMake 3.13.

Reply:
good suggestion. done